### PR TITLE
Spelling: Fixed spelling inconsistencies in WO story announcements.

### PR DESCRIPTION
**Pull request by JohnDoe50974**
## About The Pull Request
Captain Hans Naiche? Captain Hans Naich? Captain Hans Niache? WO announcements are annoying to look at because of this.

## Why It's Good For The Game
Spelling fixes improve user experience and make the game look like it was developed by people who genuinely care about it.

## Changelog

:cl: 50RemAndCounting
spellcheck: fixed a few typos and spelling inconsistencies in Captain Hans Naiche's announcements on Whiskey Outpost.
/:cl:

### CODE REVIEW

**Review by stanalbatross — APPROVED** [review]
americans when you ask them to pronounce "niche"

File: code/game/gamemodes/colonialmarines/whiskey_outpost.dm [hunk 1]
```diff
@@ -124,18 +124,18 @@
...
 		if(2)
-			marine_announcement("Captian Naich here. We've tracked the bulk of enemy forces on the move and [SSmapping.configs[GROUND_MAP].map_name] is likely to be hit before they reach the base. We need you to hold them off while we finish sending the distress call. Expect incoming within a few minutes. Godspeed, [SSmapping.configs[GROUND_MAP].map_name].", "Captain Naich, 3rd Battalion Command, LV-624 Garrison")
+			marine_announcement("Captian Naich here. We've tracked the bulk of enemy forces on the move and [SSmapping.configs[GROUND_MAP].map_name] is likely to be hit before they reach the base. We need you to hold them off while we finish sending the distress call. Expect incoming within a few minutes. Godspeed, [SSmapping.configs[GROUND_MAP].map_name].", "Captain Naiche, 3rd Battalion Command, LV-624 Garrison")
```
Comment (fira) [inline code/game/gamemodes/colonialmarines/whiskey_outpost.dm:138]: Missed one here at start

**Review by fira — APPROVED** [review]
Probably in good part my fault, i guess i copied the name typo everywhere

If you want to improve the text itself feel free, like you say about "caring about it", the real story is half of that text was indeed something I added hastily between live tests to serve as filler to try and keep the ungas waiting during game start -- buildup etc rather than just dumping it all and waiting 20 minutes.

I'm sure monkey would love you if you could rewrite it to something cooler.

File: code/game/gamemodes/colonialmarines/whiskey_outpost.dm (hunk 1 above)
Comment (JohnDoe50974) [reply code/game/gamemodes/colonialmarines/whiskey_outpost.dm:138]: Forgot about this, thanks for pointing it out. Fixed.

**Pull request merged**